rls-quick-start-guide.md

```diff
@@ -0,0 +1,51 @@
+# RLS Quick-Start Guide
+
+Hello, RLS members new and old! Welcome to the Rands Leadership Slack quick-start guide 🙂 
+
+RLS is a big community! The last time Rands made an #announcement, RLS membership was the same size as East Peoria, Illinois with 22,620 people. As RLS continues to grow, it can be harder to figure out where to get started. This quick-start guide is intended to give you that starting point to exploring and participating in this community. 
+
+And please be assured that we do want you - _yes, you!_ - to participate. You are not too junior, or too new, or not leader-y enough to contribute. Our community benefits from being able to discuss and learn from all of our perspectives and experiences, and it's what we love about being in RLS.
+
+## Code of Conduct
+
+Please make sure you read and understand the RLS [Code of Conduct](https://github.com/randsleadershipslack/documents-and-resources/blob/main/code-of-conduct.md).
```

Comment:
```suggestion
Please make sure you read and understand the RLS [Code of Conduct](https://github.com/randsleadershipslack/documents-and-resources/blob/main/code-of-conduct.md). We take it and respect quite seriously!
```

Reply:
@AldenPeterson Good addition! I would just re-word to "We respect it and take it quite seriously!".

Reply:
Added!